### PR TITLE
add rrapply via rcran skeleton

### PR DESCRIPTION
**Pull request by simonhtausch**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by simonhtausch** [comment]
@conda-forge/staged-recipes is this ready to merge?

**Comment by simonhtausch** [comment]
@conda-forge/staged-recipes is this ready to merge?

**Comment by simonhtausch** [comment]
@conda-forge-admin, please ping team

**Comment by cbrueffer** [comment]
Hi @simonhtausch , please re-generate the recipe with `https://github.com/bgruening/conda_r_skeleton_helper`. It'll generate a clean recipe that'll just need the license fixed to be SPDX conforming..

**Comment by cbrueffer** [comment]
Not sure what happened to the maintainers section, it should have added one with `conda-forge/r` in it. Could you add that one as additional maintainer?

**Comment by simonhtausch** [comment]
> Hi @simonhtausch , please re-generate the recipe with `https://github.com/bgruening/conda_r_skeleton_helper`. It'll generate a clean recipe that'll just need the license fixed to be SPDX conforming..

Hi! Thanks for the hint. The recipe generated by that script looked the same and the extra/maintainer section was missing even though I adapted the extra.yaml file. Not sure what went wrong and whether something is still missing from the recipe. I fixed the linting issues manually.

**Comment by cbrueffer** [comment]
Yeah, The recipe doesn't look quite right. Are you sure that's the one generated by the script I pointed to?

**Comment by simonhtausch** [comment]
> Basically it should look similar to e.g. https://github.com/conda-forge/r-tidybf-feedstock/blob/master/recipe/meta.yaml

It does look much alike to me at least, what would you say is wrong? The file is definitely produced by the script by björn grüning, although it terminates with an error:
```
$ bash run.sh 
Processing rrapply
Adding in variants from internal_defaults
INFO:conda_build.variants:Adding in variants from internal_defaults
Parsing input package rrapply:
.. name: rrapply location: None new_location: /home/simon/conda_r_skeleton_helper/r-rrapply
Making/refreshing recipe for rrapply
Tip: install CacheControl and lockfile (conda packages) to cache the CRAN metadata
Fetching main index from https://cran.r-project.org
/home/simon/miniconda3/lib/python3.6/site-packages/urllib3/connectionpool.py:986: InsecureRequestWarning: Unverified HTTPS request is being made to host 'cran.r-project.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
/home/simon/miniconda3/lib/python3.6/site-packages/urllib3/connectionpool.py:986: InsecureRequestWarning: Unverified HTTPS request is being made to host 'cran.r-project.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
Downloading source from https://cran.r-project.org/src/contrib/rrapply_1.2.1.tar.gz
No hash (md5, sha1, sha256) provided for source-rrapply_1.2.1.tar.gz.  Source download forced.  Add hash to recipe to use source cache.
WARNING:conda_build.source:No hash (md5, sha1, sha256) provided for source-rrapply_1.2.1.tar.gz.  Source download forced.  Add hash to recipe to use source cache.
Reading package metadata from /home/simon/miniconda3/conda-bld/src_cache/source-rrapply_1.2.1_d1c63e2b09.tar.gz
Writing recipe for rrapply
--dirty flag and --keep-old-work not specified. Removing build/test folder after successful build/test.

INFO:conda_build.config:--dirty flag and --keep-old-work not specified. Removing build/test folder after successful build/test.

Traceback (most recent call last):
  File "run.py", line 88, in <module>
    with open(meta_fname, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'rrapply/meta.yaml'
```

**Comment by dbast** [comment]
Try with r-rrapply put into packages.txt of conda_r_skeleton_helper (instead of rrapply). That works here.

File: recipes/rrapply/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    - {{ native }}gcc-libs         # [win]
+
+test:
+  commands:
+    - $R -e "library('rrapply')"           # [not win]
+    - "\"%R%\" -e \"library('rrapply')\""  # [win]
+
+about:
+  home: https://jorischau.github.io/rrapply/, https://github.com/JorisChau/rrapply
+  license: GPL-2.0-or-later
```
Comment (dbast) [inline recipes/rrapply/meta.yaml:44]:
```suggestion
  license: GPL-3.0-only
```

cran meta data tells `# License: GPL-3` which is the spdx short `GPL-3.0-only`

**Comment by dbast** [comment]
@simonhtausch Thanks!

**Comment by simonhtausch** [comment]
> @simonhtausch Thanks!

Thanks to you for the quick help!

**Pull request merged**